packages/serverless/src/domain/use-case/collection.ts

```diff
@@ -49,11 +55,21 @@ export class Collection {
     // Actor must have write permission of collection to create index
     if (actorPermission.write) {
       // Validate that all keys in the index exist in the schema
-      const indexPossible = metadataCollection.schema.map(
-        (e) => `document.${e.name}.value`
-      );
+      const indexPossible = metadataCollection.schema.map((sc) => sc.name);
```

Comment:
What's `sc`?

Reply:
it's schema. I avoid shadow the schema, but the schema from metadataCollection.schema it can't be shadowed. I changing now